pages/models.py

```diff
@@ -1272,7 +1272,8 @@ def get_context(self, *args, **kwargs):
 class GroupPage(RoutablePageMixin, Page):
     @route(r'^$')
     def index_view(self, request):
-        groups = Group.objects.all().order_by('group_type')
+        # Order is set by integer value in GROUP_TYPES tuple in models
+        groups = Group.objects.all().order_by('-group_type')
```

Comment:
Why add minus sign? Was it wrong before?

Reply:
Default sort was descending for whatever reason.